### ISSUE TITLE
Document need for `adaptive=True` option for `VTKFile`

### ISSUE THREAD

**Issue by jwallwork23** [mesh-adaptation/mesh-adaptation-docs#65]
This isn't obvious and has tripped up at least one user.

**Comment by ddundo** [comment]
Good idea. I was tripped up by this in the past as well. But maybe it would be better to open a PR to Firedrake and make the error more instructive? I could take that on :)
